django-backend/fecfiler/openfec/views.py

```diff
@@ -50,15 +50,17 @@ def retrieve_recent_f1(committee_id):
     First checks the realtime enpdpoint for a recent F1 filing.  If none is found,
     a request is made to a different endpoint that is updated nightly.
     The realtime endpoint will have more recent filings, but does not provide
-    filings older than 6 months. The nightly endpoint keeps a longer history"""
+    filings older than 6 months. The other endpoint returns a committee's current data
+    informed by their most recent F1 and F2 filings."""
     headers = {"Content-Type": "application/json"}
     params = {
         "api_key": settings.FEC_API_KEY,
         "committee_id": committee_id,
-        "sort": "-receipt_date",
-        "form_type": "F1",
     }
-    endpoints = [f"{settings.FEC_API}efile/filings/", f"{settings.FEC_API}filings/"]
+    endpoints = [
+        f"{settings.FEC_API}efile/form1/",
```

Comment:
can you change this to ```f"{settings.FEC_API}efile/form1/?sort=-load_timestamp```
there can be multiple versions of the same f1, and we need the most recent.

Reply:
Sorry, I didn't see this.

If you look at the [https://api.open.fec.gov/developers/#/efiling/get_v1_efile_filings_](docs), the default sorting is `-receipt_date` which should satisfy that requirement.